app/baseapp.go

```diff
@@ -61,10 +63,10 @@ func NewBaseApp(
 		name:             name,
 		db:               db,
 		cms:              store.NewStore(db),
-		router:           NewRouter(),
-		queryRouter:      NewQueryRouter(),
+		router:           baseapp.NewRouter(),
+		queryRouter:      baseapp.NewQueryRouter(),
 		grpcQueryRouter:  NewGRPCQueryRouter(),
```

Comment:
Related to the above comment, is there a problem with baseapp.NewGRPCQueryRouter?

Reply:
See above comment.